packages/trader/src/Modules/Contract/Components/InfoBox/info-box-longcode.tsx

```diff
@@ -26,8 +27,8 @@ const InfoBoxLongcode = ({ contract_info }) => {
                     {contract_info.longcode}
                 </Text>
                 {` `}
-                {contract_info.longcode.length > max_longcode_length && (
-                    <Text as='a' href='#' size='xs' onClick={handleToggle} c>
+                {contract_info?.longcode && contract_info.longcode.length > max_longcode_length && (
```

Comment:
```suggestion
                {contract_info?.longcode?.length > max_longcode_length && (
```

Reply:
<img width="1114" alt="Screenshot 2023-09-15 at 9 24 59 AM" src="https://github.com/binary-com/deriv-app/assets/121025168/569cdd87-9d93-4fa7-b73a-bbe2d850c46b">

Reply:
TS show error because we compare something which potentially can be undefined with number. The error can be removed by type conversion:
`Number(contract_info.longcode?.length) > max_longcode_length`
Should I convert to number or leave check (`contract_info?.longcode && contract_info.longcode.length > max_longcode_length`) ?

Reply:
sonarcloud is ok with the way it is, so let's leave it :)